Source/WebKit/Shared/WebPushDaemonConnectionConfiguration.h

```diff
@@ -36,6 +36,7 @@ struct WebPushDaemonConnectionConfiguration {
     String bundleIdentifierOverride;
     String pushPartitionString;
     std::optional<WTF::UUID> dataStoreIdentifier;
+    bool declarativeWebPushEnabled;
```

Comment:
You probably want to update these construction sites:

* `WebPushD::WebPushDaemonConnectionConfiguration WebsiteDataStoreConfiguration::webPushDaemonConnectionConfiguration`: I don't think there is an error here since uniform initialization should initialize this field with the default value of false but it's still probably good to be explicit there.
* `-[_WKWebPushDaemonConnection initWithConfiguration:]`: same as above 
* `Connection::sendAuditToken()` in `WebPushToolConnection.mm`: this should be updated

Reply:
I can just add the default initializer to the struct, actually.